### PR TITLE
add where to start paragraph and update what is Raiden

### PR DESCRIPTION
**Pull request by czepluch**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -6,8 +6,11 @@ This is the documentation for the |version| version of the `Raiden Network <http
 
 What is the Raiden Network?
 ---------------------------
-The Raiden Network is an off-chain scaling solution for performing ERC20-compliant token transfers on the Ethereum blockchain. It is Ethereum's version of Bitcoin's Lightning Network, enabling near-instant, low-fee, scalable micropayments.
+The Raiden Network is an off-chain scaling solution, enabling near-instant, low-fee and scalable payments. It’s complementary to the Ethereum blockchain and works with any ERC20 compatible token. The Raiden project is work in progress. Its goal is to research state channel technology, define protocols and develop reference implementations.
 
+Where to start?
+---------------
+If this is your first encounter with the Raiden Network it is recommended to first take a look at the :doc:`Installation Guide<overview_and_guide`. Once a Raiden node is up and running you can go try out :doc:`the Web UI<webui_tutorial>`. If you want a more in depth guide on how to start using the Raiden API you should check out the :doc:`Getting started with the Raiden API<api_walkthrough>` guide.
```
Comment (palango) [inline docs/index.rst]: There's a missing **>** after `Installation Guide<overview_and_guide`

Comment (czepluch) [reply docs/index.rst]: argh, dang it.

Comment (czepluch) [reply docs/index.rst]: fixed

Comment (palango) [reply docs/index.rst]: Easiest PR of the day, and then hiding this 🤣

**Pull request merged**